thai/massive_testing_can_stop_coronavirus.md

```diff
@@ -0,0 +1,38 @@
+
+ในกรณีของโรคโควิด-19 อันเป็นโรคระบาดของไวรัสโคโรนาที่เริ่มขึ้นในเมืองอู่ฮั่น มีวิธีการตรวจสอบดีเอ็นเอโดยใช้ตัวอย่างจากช่องจมูกหรือลำคอ ซึ่งสามารถระบุตัวผู้ติดเชื้อได้รวดเร็วเพียงพอที่จะลดอัตราการแพร่เชื้อได้อย่างมาก หากแยกผู้ที่มีผลตรวจเป็นบวกออกไปได้ การตรวจในปัจจุบันต้องใช้เวลาหลายวันกว่าจะได้ผลลัพธ์ การตรวจที่รวดเร็วกว่านี้อยู่ในขั้นตอนการพัฒนา
+
+ในช่วงต้นของการระบาด จำนวนชุดทดสอบนั้นมีจำกัดเป็นอย่างยิ่ง จึงจำเป็นต้องใช้วิธีแยกกักทั้งชุมชน จีนใช้วิธีนี้ร่วมกับการตามรอยเส้นทางการพบปะติดต่อในกลุ่มคนขนาดใหญ่ (670,000 คน) [2] และประสบความสำเร็จเป็นอย่างยิ่ง จากนั้นเกาหลีใต้ก็ได้ใช้วิธีปิดเมือง [3] ร่วมกับการตรวจหาเชื้อในวงกว้าง ซึ่งรวมถึงการอำนวยความสะดวกให้ประชาชนขับรถเข้ามาแวะตรวจได้ [4] และเมื่อไม่นานมานี้ ก็มีหลักฐานบ่งชี้ว่าการระบาดในเกาหลีใต้อยู่ในสถานะที่ควบคุมได้แล้ว [5]
+
+ขณะนี้ หลายประเทศทั่วโลก รวมถึงสหรัฐอเมริกา ยังมีชุดตรวจไม่เพียงพอที่จะทำการทดสอบในวงกว้าง และนี่ก็คือข้อจำกัดในการใช้วิธีนี้ ถึงกระนั้นก็เป็นไปได้ที่จะพัฒนาวิธีตรวจสอบที่รวดเร็วและมีราคาถูกเพื่อนำไปใช้ในวงกว้าง และลดความจำเป็นในการใช้วิธีการอื่น ๆ เช่นการปิดเมือง เมื่อมีชุดตรวจจำนวนมาก การตรวจที่จำเพาะเจาะจงในวงกว้าก็จะช่วยให้เราบรรลุเป้าหมายที่ต้องการในการหยุดยั้งโรคระบาดได้
+
+แล้วเราควรจะทำอย่างไรบ้าง การชะลอหรือหยุดยั้งโรคระบาดไวรัสโคโรนาสามารถทำได้หลายวิธีร่วมกัน บุคคล ครอบครัว และชุมชนควรระมัดระวังตนให้มาก เพื่อหลีกเลี่ยงการสัมผัสกับเชื้อ และลดโอกาสที่ตนเองและผู้อื่นจะติดเชื้อ ในขณะเดียวกัน หน่วยงานสาธารณสุขควรเร่งกำลังการตรวจหาเชื้อ ด้วยการกำหนดคุณสมบัติขั้นต่ำและเปิดทางให้หลาย ๆ พื้นที่ทางภูมิศาสตร์เข้าถึงการตรวจได้ หากเป็นไปได้ บริษัทหรือองค์กรพัฒนาเอกชนอาจให้บริการตรวจในสถานที่ที่สะดวกต่อประชาชน โดยอาจเป็นการตรวจตามบ้าน เช่นเดียวกับในช่วงท้ายๆ ของการระบาดในประเทศจีน [6] เพื่อระบุผู้ติดเชื้อให้รวดเร็ว
+
+### เอกสารอ้างอิง
+- [How community response stopped ebola](https://necsi.edu/how-community-response-stopped-ebola)
+- [Report of the WHO-China Joint Mission on Coronavirus Disease 2019 (COVID-19)](https://www.who.int/docs/default-source/coronaviruse/who-china-joint-mission-on-covid-19-final-report.pdf)
+- [Daegu in Lockdown as Coronavirus Infections Soar](http://english.chosun.com/site/data/html_dir/2020/02/24/2020022401353.html)
+- [South Korea pioneers coronavirus drive-through testing station](https://www.cnn.com/2020/03/02/asia/coronavirus-drive-through-south-korea-hnk-intl/index.html)
+- [BREAKING: Coronavirus Update. Significant decline in daily new cases in South Korea--positive sign of gaining control.](https://twitter.com/yaneerbaryam/status/1235734017699430401?s=20)
+- [China Goes Door to Door in Wuhan, Seeking Infections](https://www.courthousenews.com/china-goes-door-to-door-in-wuhan-seeking-infections/)
```

Comment:
@aprossi @futtetennista I notice that the English source is missing these urls

Reply:
yes, they were in version 42, not 43 it seems, copy paste might have removed the old markdown links?